### PR TITLE
[CDAP-17103] Fix showing banner in studio when backend is down

### PR DESCRIPTION
**Pull request by yukiej**
JIRA: https://issues.cask.co/browse/CDAP-17103
Build: https://builds.cask.co/browse/CDAP-URUT287

### CODE REVIEW

**Comment by yukiej** [comment]
Screen Shot 2020-07-21 at 6 39 08 PM

File: cdap-ui/app/cdap/components/LoadingIndicator/index.js [hunk 1]
```diff
@@ -40,7 +40,10 @@ export default class LoadingIndicator extends Component {
 
   componentDidMount() {
     this.loadingIndicatorStoreSubscription = LoadingIndicatorStore.subscribe(() => {
-      if (location.pathname.indexOf('/cdap/administration') !== -1) {
+      if (
+        location.pathname.indexOf('/cdap/administration') !== -1 ||
```
Comment (jennac3) [inline cdap-ui/app/cdap/components/LoadingIndicator/index.js]: nit: we may also use `includes` instead of `indexOf`

**Review by jennac3 — APPROVED** [review]

File: cdap-ui/app/cdap/components/LoadingIndicator/index.js (hunk 1 above)
Comment (yukiej) [reply cdap-ui/app/cdap/components/LoadingIndicator/index.js]: Done!

**Review by elfenheart — APPROVED** [review]
👍  LGTM

**Review by ajainarayanan — APPROVED** [review]

**Pull request closed without merging**